content/blog/tidymodels-errors-q4/index.Rmd

```diff
@@ -0,0 +1,229 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-errors-q4
+title: "Three ways errors are about to get better in tidymodels"
+date: 2023-11-13
```

Comment:
Slated with hopes that the linked recipes PR will be merged by Monday, though we can push back if needed.

Reply:
Recipes PR has been merged! Bumped to this Friday, the 10th.